students/tri_nguyen/lesson05/mailroom.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  09-Mar-2018
+# ------------------------------------------- #
+
+# Import stuff
+from operator import itemgetter
+import time
+
+# Get local time
+local_time = time.localtime()
+
+# Declare variables and data structure
+
+donor_dict = { #  [0] == total donation, [1] == number of gifts, [2] == average
+
+    'William Gates, III': [653784.49, 2, 326892.24],
+    'Mark Zuckerberg': [16396.10, 3, 5465.37],
+    'Jeff Bezos': [877.33, 1, 877.33],
+    'Paul Allen':[708.42, 3, 236.14],
+    'Tri Nguyen': [100.00, 1, 100.00]
+    }
+
+    
+# Declare functions
+
+
+def display_menu():
+    ''' this function displays a menu
+        of 3 actions '''
+
+    print(''' Select one of the actions below:
+
+    1. Send a Thank You
+    2. Create a Report
+    3. Send letters to everyone
+    4. Quit
+
+    ''')
+
+
+def display_donor_list():
+    ''' this function displays list of donors
+        from donor_dict '''
+
+    donor_name = enumerate([donor for donor in donor_dict])
+    for idx, name in donor_name:
+        print(idx + 1, '-', name)
```

Comment:
-> `for idx, name in enumerate(donor_dict):`

dictionaries iterate over keys by default